### PR TITLE
aligned with shoestring

### PR DESCRIPTION
**Pull request by Okami-**
- Added max-width to container.
- Moved shoestring html markup 
- Changed classes in javascript to new markup
- Added light-blue color class instead

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,7 +18,19 @@
 
 </head>
 <body>
-<div id="ucsf-banner-nav" class="logo">
+<div id="top-header-region" class="logo">
```
Comment (stopfstedt) [inline index.html]: this is making it worse. if anything, we should revert to a top-level class name that can be assumed unique in any context and carries some valuable semantic information. 
proposing instead 

``` css
<div class="ucsf-banner-nav logo">
```

File: banners.css [hunk 2]
```diff
@@ -158,3 +158,101 @@
...
+
+/* Shoestring Css */
+
+#top-header-region {
+  height: 40px;
+  overflow: hidden;
+  font-family: "Helvetica Neue", "arial", sans-serif;
+  background: #052049; }
+
+  /* line 4, /Users/richardbaker/git_repos/shoestring/src/assets/sass/ucsf/components/banner */
```
Comment (stopfstedt) [inline banners.css]: please lose these comments. thanks.

File: banners.css [hunk 3]
```diff
@@ -158,3 +158,101 @@
...
+  #top-header-region .top-header-container {
```
Comment (stopfstedt) [inline banners.css]: nein. also, why leave all the other styles above in there then?

**Comment by Okami-** [comment]
LOL I'm pretty much rewriting this. I'll clean up the old code and comments.

**Comment by Okami-** [comment]
I know the app is small but I moved the logic to a separate file.

**Comment by johnkealy** [comment]
I always thought the banner should be an ID... avoids so many problems.

**Comment by stopfstedt** [comment]
@Okami- please get rid of underlining.

selection_017

**Comment by stopfstedt** [comment]
@Okami-  - also, the JS event handling on the demo page is broken, please push `banner.js` up as well.

**Comment by stopfstedt** [comment]
almost there buddy. will need the SVG logo back for anything non-IE8-.

**Comment by stopfstedt** [comment]
manually merged. all good.

**Pull request closed without merging**